samwise.js

```diff
@@ -1,10 +1,10 @@
-/**
```

Comment:
SCRIPT-You did a great job handling all the possible error states in the UI. It enhances the user experience significantly.